mail_service/mail_views.py

```diff
@@ -11,6 +11,14 @@ async def send_message(body, mail_to):
     message["From"] = "uacinema.nure@localhost"
     message["To"] = "uacinema.nure@gmail.com"
     message["Subject"] = "Activity on UA Cinema"
+    message.set_content(body)
+
+    try:
+        async with aiosmtplib.SMTP(hostname="smtp.gmail.com", port=465, use_tls=True) as smtp: # auto connect/disconnect
+            await smtp.login('test@gmail.com', 'password')  # gmail credentials
+            await smtp.send_message(message)
+    except (aiosmtplib.SMTPAuthenticationError, aiosmtplib.SMTPException) as e:
+        print(f"Exception raised {e}, check  credentials or email service configuration")
```

Comment:
message from and to are hardcoded. implement sending message to redis from backend and parse it here appropriately